src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2750,13 +2993,47 @@ static void _updateComposite(SvgNode* node, SvgNode* root)
 }
 
 
+static void _updateCssStyle(SvgNode* doc, SvgNode* cssStyle)
```

Comment:
the last update - all nodes has to be checked, not only the ones with class attrib (e.x. style set for all circles will be applied here).